docs/guides/mobile-plugin-gma-ios.md

```diff
@@ -29,9 +29,13 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-1. [Google Mobile Ads SDK v10.7.0+](https://developers.google.com/admob/ios/rel-notes)
-1. [UID2 SDK for iOS v0.2.0](../sdks/uid2-sdk-ref-ios.md)
-1. UID2 GMA Plugin for iOS v0.2.0
+1. Google Mobile Ads SDK v10.7.0 or later:
+   - [SDK](https://developers.google.com/admob/ios)
+   - [Release Notes](https://developers.google.com/admob/ios/rel-notes)
+1. UID2 SDK for iOS v0.2.0:
+   - [SDK](https://github.com/IABTechLab/uid2-ios-sdk)
+   - [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
```

Comment:
Not needed.  See comment in `docs/guides/mobile-plugin-gma-android.md`

Reply:
again -- I'd like to include a link to the SDK doc in the plug-in doc... it isn't mentioned elsewhere in the doc. Is it a problem? LMK if so and I'll take it out.

Reply:
Resolved above.